core/src/main/java/org/apache/iceberg/TableOperations.java

```diff
@@ -30,6 +30,8 @@
  */
 public interface TableOperations {
 
+  String tablePath();
```

Comment:
Why not use `table.location()`? You can access that using `ops.current().location()`.

I'm really reluctant to add more methods to this interface since it is an important place where people can plug in their own customizations.

Reply:
I guess table location is also fine. We can parse out table name and db name from it.